test/libsolidity/smtCheckerTests/operators/assignment_contract_member_variable.sol

```diff
@@ -0,0 +1,29 @@
+pragma experimental SMTChecker;
+contract A {
+	int x;
+	int y;
+	function a() public {
+		require(A.x < 100);
+		A.y = A.x++;
+		assert(A.y == A.x - 1);
+		// Fails
+		assert(A.y == 0);
+		A.y = ++A.x;
+		assert(A.y == A.x);
+		delete A.x;
+		assert(A.x == 0);
+		A.y = A.x--;
+		assert(A.y == A.x + 1);
```

Comment:
Can you add one more `assert(A.y == 0)` here, or it will be too much?

Reply:
Added